apps/web/src/pages/templates/components/VariantsPage.tsx

```diff
@@ -75,7 +77,9 @@ export function VariantsPage() {
             />
           );
         })}
-        <VariantItemCard isReadonly={isReadonly} key={stepUuid} variant={step} nodeType="variantRoot" />
+        {step && variants.length > 0 && (
+          <VariantItemCard isReadonly={isReadonly} key={stepUuid} variant={step} nodeType="variantRoot" />
+        )}
```

Comment:
when no variants and no step don't render the root step in the variants list sidebar